.travis.yml

```diff
@@ -36,15 +36,11 @@ after_success:
 
 jobs:
   include:
-    - language:
-      python:
+    - language: "python"
       env:
         - TESTS="Docker"
       services:
         - docker
-      before_install:
-      before_script:
-      install:
       script:
         - docker build -f Dockerfile -t openwpm .
         - ./scripts/deploy-to-dockerhub.sh
```

Comment:
I think this is causing the failure in building the Docker image on Travis. Removing the `before_install:`, `install:`, and `before_script:` sections from the Docker job makes Travis use these sections as they were specified [above](https://github.com/mozilla/OpenWPM/blob/master/.travis.yml#L17L28) and unecessarily installs everything before building the Docker image like it does for the rest of the test jobs.

If we put these back it should be alright.

Reply:
Thank you for pointing this out! I'll revert the changes and check.

Reply:
This worked! Thanks for pointing this out.